lib/_stream_transform.js

```diff
@@ -76,7 +76,7 @@ function afterTransform(er, data) {
 
   var cb = ts.writecb;
 
-  if (!cb) {
+  if (cb === null) {
```

Comment:
This potentially could be a breaking change given that it would miss other falsy values potentially passed for `cb`